app/presenters/grade_summary_assignment_presenter.rb

```diff
@@ -121,6 +121,15 @@ def turnitin
     t.try(:[], :state) ? t : nil
   end
 
+  def vericite
```

Comment:
Other than pulling from the vericite_data method, this is the same as the turnitin item above.  These should be combined.